src/plugin/reactPlugin/2-pageTransform/toLoaderScript.js

```diff
@@ -38,6 +38,6 @@ module.exports = function toLoaderScript({ componentPath, hydrate, id, props = {
     });
   </script>`
   } else {
-    throw 'Unsupported loader'
+    return ''
```

Comment:
Helps with mapping on `render = "static"`. Simpler to handle and empty string over a needless exception 👍